src/environments/environment.prod.ts

```diff
@@ -1,3 +1,18 @@
 export const environment = {
-  production: true
+  get production() {
+    return true;
+  },
+  get name() {
+    return 'production';
+  },
+  get apis() {
+    return {
+      get questionServiceConsumerEndpoint() {
+        return 'https://question-service-ft1.azurewebsites.net/api/question';
```

Comment:
I assume this is purely for testing? If so then we need to update these so they point to the correct respective environments

Reply:
done